cockatrice/cockatrice_en@source.ts

```diff
@@ -5163,7 +5179,8 @@ Please check your shortcut settings!</source>
     <name>TabDeckEditor</name>
     <message>
         <location filename="src/tab_deck_editor.cpp" line="379"/>
-        <source>Search by card name</source>
+        <source>Search by card name (or search expressions)</source>
+        <oldsource>Search by card name</oldsource>
```

Comment:
Hmm. Do we want this `oldsource` in our file?
First time I'm seeing this. Will have to check how/if that is used in Transifex after merge.

There are only two occurences in all source.ts file, those are maybe related to the new 
`Similar-text heuristic provided 2 translation(s)` output?

Reply:
Any clue here? What do you think? @ebbit1q